TSC-Charter.md

```diff
@@ -162,12 +181,13 @@ for dissent from the consensus.
 If an agenda item cannot reach a consensus a TSC member can call for
 either a closing vote or a vote to table the issue to the next meeting.
 The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue. Simple majority wins, with the following
-exceptions, which will require the affirmative vote of two-thirds of the
-members of the TSC to pass:
+discussion will continue. Simple majority wins with exception given only to
+removal of a member from the TSC, which shall require a two-thirds majority
+vote to approve.
```

Comment:
The motivation for lowering to 51% is that it's proven hard to get 66%. It's not that there are >33% "no" votes, but that there is usually >33% "didn't vote." It's currently really hard to get enough people to vote so that we have >66% "yes" votes, even when everyone who voted did vote "yes"